### DISCUSSION TITLE
Judgements not appearing with "Aim Assist" mod

### DISCUSSION THREAD

**Discussion by ArchUsr64** [ppy/osu#16812] [Q&A] [answered]
https://user-images.githubusercontent.com/83179501/152778388-947352ad-ff66-4823-8ee9-bbdf74b7bb6e.mp4

Hit burst objects are not appearing when "Aim Assist" mod is enabled. To my knowledge there aren't any issue reports on the subject and this doesn't seem intentional.

**Comment by uzervlad** [comment, marked as answer]
[It's intentional](https://github.com/ppy/osu/blob/master/osu.Game.Rulesets.Osu/Mods/OsuModAimAssist.cs#L43-L45)

**Reply by ArchUsr64** [reply]
Followpoints will probably not work but I think having hit burst objects such as "hit100.png" should be fine and without them visual accuracy feedback is absent.

**Reply by peppy** [reply]
If you read the PR you will note that it's not implemented due to technical constraints. It may be in the future if there is enough interest.
